smac/runhistory/runhistory2epm.py

```diff
@@ -725,5 +725,5 @@ def transform_response_values(self, values: np.ndarray) -> np.ndarray:
         -------
```

Comment:
We do we not add 1 anymore?

Reply:
This aims at solving the issue #833

Reply:
Related to #833 , todo: create a separate PR for that:)